internal/services/manageddevopspools/managed_devops_pool_data_source.go

```diff
@@ -0,0 +1,381 @@
+			Type:     pluginsdk.TypeList,
+			Computed: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"organization": {
+						Type:     pluginsdk.TypeList,
+						Computed: true,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"parallelism": {
+									Type:     pluginsdk.TypeInt,
+									Computed: true,
+								},
+
+								"projects": {
+									Type:     pluginsdk.TypeList,
+									Computed: true,
+									Elem: &pluginsdk.Schema{
+										Type: pluginsdk.TypeString,
+									},
+								},
+
+								"url": {
+									Type:     pluginsdk.TypeString,
+									Computed: true,
+								},
+							},
+						},
+					},
+
+					"permission_profile": {
+						Type:     pluginsdk.TypeList,
+						Computed: true,
+						Elem: &pluginsdk.Resource{
+							Schema: map[string]*pluginsdk.Schema{
+								"kind": {
+									Type:     pluginsdk.TypeString,
+									Computed: true,
+								},
+
+								"administrator_account": {
+									Type:     pluginsdk.TypeList,
+									Computed: true,
+									Elem: &pluginsdk.Resource{
+										Schema: map[string]*pluginsdk.Schema{
+											"groups": {
+												Type:     pluginsdk.TypeList,
+												Computed: true,
+												Elem: &pluginsdk.Schema{
+													Type: pluginsdk.TypeString,
+												},
+											},
+
+											"users": {
+												Type:     pluginsdk.TypeList,
+												Computed: true,
+												Elem: &pluginsdk.Schema{
+													Type: pluginsdk.TypeString,
+												},
+											},
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+
+		"dev_center_project_resource_id": {
+			Type:     pluginsdk.TypeString,
+			Computed: true,
+		},
+
+		"identity": commonschema.UserAssignedIdentityComputed(),
+
+		"maximum_concurrency": {
+			Type:     pluginsdk.TypeInt,
+			Computed: true,
+		},
+
+		"stateful_agent_profile": {
+			Type:     pluginsdk.TypeList,
+			Computed: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"grace_period_time_span": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
+					},
+
+					"max_agent_lifetime": {
+						Type:     pluginsdk.TypeString,
+						Computed: true,
+					},
+
+					"manual_resource_predictions_profile": manualResourcePredictionsProfileSchemaComputed(),
+
+					"automatic_resource_predictions_profile": automaticResourcePredictionsProfileSchemaComputed(),
+				},
+			},
+		},
+
+		"stateless_agent_profile": {
+			Type:     pluginsdk.TypeList,
+			Computed: true,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"manual_resource_predictions_profile": manualResourcePredictionsProfileSchemaComputed(),
+
+					"automatic_resource_predictions_profile": automaticResourcePredictionsProfileSchemaComputed(),
+				},
+			},
+		},
+
+		"tags": commonschema.TagsDataSource(),
+
+		"vmss_fabric_profile": {
```

Comment:
Could you update this following our naming conventions? ([ref](https://github.com/hashicorp/terraform-provider-azurerm/blob/main/contributing/topics/reference-naming.md?plain=1#L15-L16))

```suggestion
		"virtual_machine_scale_set_fabric_profile": {
```